website/src/content/docs/docs/libraries/typespec-client-generator-core/reference/decorators.md

```diff
@@ -186,15 +188,25 @@ scalar storageDateTime extends utcDataTime;
 op test(@param @alternateType(string) date: utcDateTime): void;
 ```
 
+````typespec
+model Test {
+  @alternateType(unknown)
+  thumbprint?: string;
+
+  @alternateType(AzureLocation[], "csharp")
+  locations: string[];
+}
+} * ```
```

Comment:
here we have some format issues. Is this file generated? We should fix it.

Reply:
> here we have some format issues. Is this file generated? We should fix it.

Yes, it's generated.

Reply:
fixed the doc issue in decorators.tsp.